sell-loot.lic

```diff
@@ -18,22 +18,38 @@ class SellLoot
     Flags.add('tip-expired', 'Your tip offer to .* has expired')
 
     settings = get_settings
-    town_data = get_data('town')
+    @town_data = get_data('town')
     @actual_fucking_hometown = settings.hometown
-    @hometown = town_data[settings.hometown]
+    @hometown = @town_data[settings.hometown]
     @bankbot_name = settings.bankbot_name
     @bankbot_deposit_threshold = settings.bankbot_deposit_threshold
     @bankbot_room_id = settings.bankbot_room_id
 
+    temp_currency = @town_data[@actual_fucking_hometown]["currency"]
     sell_gems("#{settings.gem_pouch_adjective} pouch") if settings.sell_loot_pouch
 
     check_spare_pouch(settings.spare_gem_pouch_container, settings.gem_pouch_adjective) if settings.spare_gem_pouch_container
 
     sell_bundle if settings.sell_loot_bundle
 
-    return if settings.sell_loot_skip_bank
+    if settings.sell_loot_skip_bank && settings.bankbot_name
```

Comment:
I'm not a fan of this logical combo, If I have skip bank set I don't expect to be going to a bankbot. 
It seems like our options should be, Normal banking, No Banking, Bank Bot with exchange, Bank Bot without exchange. I'd kind of just like to add a `bankbot_skip_exchange: true` flag.